/var/lib/slurm/slurmd/job583995/slurm_script: 11: source: not found
Xet Storage is enabled for this repo, but the 'hf_xet' package is not installed. Falling back to regular HTTP download. For better performance, install the package with: `pip install huggingface_hub[hf_xet]` or `pip install hf_xet`
Some weights of BertForSequenceClassification were not initialized from the model checkpoint at bert-base-uncased and are newly initialized: ['classifier.bias', 'classifier.weight']
You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.
The new embeddings will be initialized from a multivariate normal distribution that has old embeddings' mean and covariance. As described in this article: https://nlp.stanford.edu/~johnhew/vocab-expansion.html. To disable this, use `mean_resizing=False`
model_name:  bert-base
model_path:  bert-base-uncased
new_token_id:  30522
Unsupervised evaluation
batch_idx:  0
batch_idx:  1
batch_idx:  2
batch_idx:  3
batch_idx:  4
batch_idx:  5
batch_idx:  6
batch_idx:  7
batch_idx:  8
batch_idx:  9
batch_idx:  10
batch_idx:  11
batch_idx:  12
batch_idx:  13
batch_idx:  14
batch_idx:  15
batch_idx:  16
batch_idx:  17
batch_idx:  18
batch_idx:  19
batch_idx:  20
batch_idx:  21
batch_idx:  22
batch_idx:  23
batch_idx:  24
batch_idx:  25
batch_idx:  26
batch_idx:  27
batch_idx:  28
batch_idx:  29
batch_idx:  30
batch_idx:  31
batch_idx:  32
Traceback (most recent call last):
  File "/home/r/roald/ELCo/scripts/emote.py", line 135, in <module>
    unsupervised_eval.unsupervised_evaluation()
  File "/home/r/roald/ELCo/scripts/unsupervised.py", line 81, in unsupervised_evaluation
    if instance[self.new_emote_config.entailment_idx] > instance[self.new_emote_config.neutral_idx] + instance[self.new_emote_config.contradiction_idx]:
IndexError: index 2 is out of bounds for axis 0 with size 2
srun: error: xgpi14: task 0: Exited with exit code 1
